example/linalg/example_svd1.f90

```diff
@@ -0,0 +1,50 @@
+! Singular Value Decomposition 
+program example_svd1
+  use stdlib_linalg_constants, only: dp
+  use stdlib_linalg, only: svd
+  implicit none
+
+  real(dp), allocatable :: A(:,:),s(:),u(:,:),vt(:,:)
+
+  ! We want to find the singular value decomposition of matrix: 
+  ! 
+  !    A = [ 3  2  2]
+  !        [ 2  3 -2]  
+  !   
+  A = transpose(reshape([ 3, 2, 2, &
+                          2, 3,-2], [3,2]))
+
+  ! Prepare arrays
+  allocate(s(2),u(2,2),vt(3,3))
+
+  ! Get singular value decomposition
+  call svd(A,s,u,vt)
+
+  ! Singular values: [5, 3]
+  print 1, '    '
+  print 1, 'S = ',s
+  print 1, '    '
+  
+  ! Left vectors (may be flipped): 
+  !     [�2/2  �2/2]
+  ! U = [�2/2 -�2/2]
+  !
+  print 1, '    '
+  print 1, 'U = ',u(1,:)
+  print 1, '    ',u(2,:)
+  
+
+  ! Right vectors (may be flipped): 
+  !     [�2/2    �2/2      0]
+  ! V = [1/�18 -1/�18  4/�18]
+  !     [ 2/3    -2/3   -1/3]
+  !
+  print 1, '    '
+  print 1, '    ',vt(1,:)
+  print 1, 'VT= ',vt(2,:)
+  print 1, '    ',vt(3,:)
+  print 1, '    '
+  
+  1 format(a,*(1x,f12.8))
```

Comment:
I suggest to create  a `parameter` `character` instead of using this notation.

Reply:
I agree on the modern version: [8f35e98](https://github.com/fortran-lang/stdlib/pull/808/commits/8f35e9856aaba14f1e715d647896d89037bddaed)